cmd/start.go

```diff
@@ -104,5 +104,6 @@ This command will start a stack and run it in the background.
 
 func init() {
 	startCmd.Flags().BoolVarP(&startOptions.NoRollback, "no-rollback", "b", false, "Do not automatically rollback changes if first time setup fails")
+	startCmd.Flags().StringVarP(&startOptions.FirstEvent, "first-event", "f", "0", "Specify the starting block for event processing (default: 0)")
```

Comment:
A few things:
- Move this to the init command
- Would suggest renaming to `--from-block`  and I wouldn't put a shortcut for this one of `-f` 
- I would make this super obvious that is only applies to multiparty as a contract listener gets created for that contract... It might be better as well to not create the contract listener at all

```suggestion
	startCmd.Flags().StringVarP(&startOptions.FirstEvent, "--from-block", "f", "0", "Specify the starting block for event processing (default: 0)")
```